.travis.yml

```diff
@@ -16,11 +16,13 @@ before_script:
   - if [ "$TRAVIS_RUST_VERSION" == "nightly" ]; then
         rustup component add --toolchain nightly rustfmt-preview;
```

Comment:
I'm not sure but isn't `rustfmt` currently preferred over the older `rustfmt-preview`.

Reply:
I think I just copied some old travis script so I'll change that.

Reply:
Apparently if you look at https://static.rust-lang.org/dist/channel-rust-nightly.toml, it renames rustfmt to rustfmt-preview.

```
oliver@canopus:~/repos/dominion$  curl -s https://static.rust-lang.org/dist/channel-rust-nightly.toml | grep -A1 rename | grep rustfmt
[renames.rustfmt]
to = "rustfmt-preview"
```